CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-﻿# 2.20.0
+﻿# 2.20.1
```

Comment:
this should probably be a feature revision (minor release) since its a new thing

Reply:
There were conflicts ... jest resolved them

Reply:
it should still be a minor and not patch revision :)

Reply:
also, there's a lint error in your test case